src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -49,12 +49,24 @@
       </c:forEach>
     </select>
   </div>
-  <div class="${divClassType}<c:if test="${questionWithResponses.question.questionTypeConstsum}"> width-auto</c:if>">
-    ${response.submissionFormHtml}
-    <c:if test="${response.existingResponse}">
-      <input type="hidden"
-          name="<%= Const.ParamsNames.FEEDBACK_RESPONSE_ID %>-${questionWithResponses.question.qnIndx}-${response.responseIndx}"
-          value="<c:out value="${response.responseId}"/>">
-    </c:if>
-  </div>
+  <c:choose>
+    <c:when test="${questionWithResponses.question.questionTypeConstsum}">
```

Comment:
To me, I can accept the solution :)

Maybe you could improve the arrangement to something like this?
![image](https://user-images.githubusercontent.com/13977235/35179501-9de1dfb0-fdd6-11e7-8676-dcf49362f4b1.png)

To me, the width of the shouldn't be more than the content.
![image](https://user-images.githubusercontent.com/13977235/35179515-cc611130-fdd6-11e7-8abe-41016bbcbf0c.png)

Reply:
I have edited the files accordingly. Should I proceed to make similar changes for rank options questions as per the discussion in #7899 ?